Mailbox/Person.cs

```diff
@@ -1,4 +1,52 @@
-﻿namespace Mailbox
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Mailbox
 {
+    public class Person : IEquatable<Person>
+    {
+        public string LastName { get; set; }
+        public string FirstName { get; set; }
+
+        public Person(string lastName, string firstName)
+        {
+            if(lastName.Equals(null) || firstName.Equals(null))
```

Comment:
This is not the correct way to check for null. If `lastName` is null, then this will throw a NullReferenceException not an ArgumentException.
```suggestion
            if(lastName is null || firstName is null)
```